tests/test_functional/fastapi_chat_completion_openai/server_openai.py

```diff
@@ -36,9 +35,16 @@ def create_agent(dto: CreateAgentDto):
         api_key=dto.api_key,
     )
 
-
 bridge = LangchainOpenaiApiBridgeFastAPI(app=app, agent_factory_provider=create_agent)
 bridge.bind_openai_chat_completion(prefix="/my-custom-path")
 
+def custom_event_handler(event):
```

Comment:
Existing tests should not be changed. 
Reason: We do not want to alter the use case of not having a "custom event handler".  We still need to test that the library works without it.

A new integration test should be made for this specific use case.